gdUnit3-examples/RoomDemo3D/src/rooms/RoomWithDoor.gd

```diff
@@ -0,0 +1,21 @@
+class_name RoomWithDoor
+extends Spatial
+
+
+signal door_closed(door_id)
+signal door_opened(door_id)
+
+func _ready():
+	Input.set_mouse_mode(Input.MOUSE_MODE_CAPTURED)
+
+func _process(_delta):
+	
+	if Input.is_action_just_pressed("input_exit"):
```

Comment:
Rename input keys to demo-specific names to avoid conflicts with keys of the same name in other projects